drs3/pubsub/subscribe.py

```diff
@@ -37,6 +40,22 @@ def process_file_staged_message(message: dict, config):
     handle_staged_file(message=message, config=config)
 
 
+def process_file_registered_message(
+    message: dict,
+    publish_object_registered: Callable[[DrsObjectInitial, Config], None],
```

Comment:
Here I think it is not necessary to pass the publish function as an argument. This `process_file_registered_message` can just directly forward the imported `publish_drs_object_registered` to the `handle_registered_file` function.

The only reason why we are doing this function via argument for the `handle_registered_file` is that a core function shouldn't be aware of where the publish function is located. But all functions in the `pubsub` module (including the `process_file_registered_message`) are allowed to directly import and use any publish functions.